Sources/BBCodeParser.php

```diff
@@ -4195,7 +4199,7 @@ protected function transformToHtml(array $tag, array $params): void
 			}
 
 			// We want $1 to be the content, and the rest to be csv.
-			$data = explode(',', ',' . substr($this->message, $this->pos1, $pos2 - $this->pos1));
+			$data = explode(',', ',' . substr($this->message, $this->pos1, (int) ($pos2 - $this->pos1)));
```

Comment:
Casting the result of the expression to int is unnecessary. The result of the arithmetic operation will already be an integer.